rocketpy/plots/rocket_plots.py

```diff
@@ -335,6 +337,30 @@ def draw(self, vis_args=None):
         ax.scatter(
             nozzle_position, 0, label="Nozzle Outlet", color="brown", s=10, zorder=10
         )
+
+        if isinstance(self.rocket.motor, (SolidMotor, HybridMotor)):
+            chamber = self.rocket.motor.plots._generate_combustion_chamber(
+                translate=(nozzle_position, 0), csys=self.rocket._csys
+            )
+            ax.add_patch(chamber)
+            grains = self.rocket.motor.plots._generate_grains(
+                translate=(nozzle_position, 0), csys=self.rocket._csys
+            )
+            for grain in grains:
+                ax.add_patch(grain)
+
+        elif isinstance(self.rocket.motor, LiquidMotor):
```

Comment:
Only to make sure I understand the situation, the reason the draw is broken up in pieces here instead of directly calling the `motor.draw()` is because the final result would be too cluttered?